src/styles/PowerSelect.scss

```diff
@@ -263,3 +259,18 @@
     width: 100%;
   }
 }
+
+/************************************************************************
+Tether specific style changes to suport auto reposition of drop down menu
+*************************************************************************/
+.tether-target-attached-top {
```

Comment:
this might apply to other components using tether. Can we target this to 

```scss
.PowerSelect__Tether {
  &.tether-target-attached-top {
  }
  &.tether-target-attached-bottom {
  }
}
```